src/script/page/LeftSidebar/panels/Conversations/ConversationHeader/ConversationHeader.tsx

```diff
@@ -113,8 +113,6 @@ export const ConversationHeader = ({
             css={button}
           >
             <Icon.PlusIcon />
-
-            {t('conversationGroupCreate')}
```

Comment:
Maybe we can include it as button's `title` prop?

Reply:
I went with a more relevant string (the previous one is just "Group")
![image](https://github.com/wireapp/wire-webapp/assets/78490891/4b10f377-8afd-4084-b3fa-93379b872bd4)